geowebcache/wms/src/test/java/org/geowebcache/config/GetCapabilitiesConfigurationTest.java

```diff
@@ -21,85 +29,203 @@
 
 import com.google.common.collect.Sets;
 
-public class GetCapabilitiesConfigurationTest {
-    
-    @Before
-    public void setUp() throws Exception {
-    }
-    
-    @Test
-    public void testDelegateInitializingLayers() throws Exception {
-        GridSetBroker broker = new GridSetBroker(false, false);
-        String url = "http://test/wms";
-        String mimeTypes = "image/png";
-        
-        final WebMapServer server = createMock(WebMapServer.class);
-        WMSCapabilities cap = createMock(WMSCapabilities.class);
-        WMSRequest req = createMock(WMSRequest.class);
-        OperationType gcOpType = createMock(OperationType.class);
-        XMLConfiguration globalConfig = createMock(XMLConfiguration.class);
-        Capture<TileLayer> layerCapture = new Capture<TileLayer>();
-        
-        GetCapabilitiesConfiguration config = 
-                new GetCapabilitiesConfiguration(broker, url, mimeTypes, "3x3", "false"){
+    public class GetCapabilitiesConfigurationTest {
```

Comment:
You seem to have indented the entire class here, it's making it rather hard to evaluate the specific changes.

Reply:
The indentation fix is done in it's own commit so the other commits can be used individually to see relevant changes.